src/scripts/shards/Shards.ts

```diff
@@ -124,6 +124,10 @@ class Shards implements Feature {
         }
     }
 
+    public static image(type: number): string {
+        return `assets/images/shards/${type}.png`;
+    }
+
```

Comment:
Will this need to be updated with #1197 being merged?

Reply:
Yep, I've merged the changes